### PR TITLE
doc: Translate Getting Started

### PR DESCRIPTION
**Pull request by fabianlee1211**
This PR is ready for review!

I kept the following words untranslated:
`ecosystem`
`widget`

And I am not sure about translating
`dynamic` to `可變`

### CODE REVIEW

File: content/docs/getting-started.md [hunk 1]
```diff
@@ -18,109 +18,109 @@ redirect_from:
...
 
-### Online Playgrounds {#online-playgrounds}
+### 線上體驗 {#online-playgrounds}
 
-If you're interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world), [CodeSandbox](https://codesandbox.io/s/new), or [Glitch](https://glitch.com/edit/#!/remix/starter-react-template).
+如果你有興趣嘗試 React，你可以使用一些線上編輯器，包括 [CodePen](codepen://hello-world)、[CodeSandbox](https://codesandbox.io/s/new)，或 [Glitch](https://glitch.com/edit/#!/remix/starter-react-template)，實作一個 Hello World 的範例。
 
-If you prefer to use your own text editor, you can also [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so we'd only recommend using this for simple demos.
+如果你偏向使用自己的編輯器，你可以[下載此 HTML 文件](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html)進行修改，並從本地文件系統裏把它打開到瀏覽器。這會進行一個緩慢的 runtime 程式碼轉換，因此我們建議只把這文件用作簡單的示範。
```
Comment (neighborhood999) [inline content/docs/getting-started.md]:
```suggestion
如果你偏向使用自己的編輯器，你可以[下載此 HTML 文件](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html)進行修改，並從本機文件系統裡把它打開到瀏覽器。這會進行一個緩慢的 runtime 程式碼轉換，因此我們建議只把這文件用作簡單的示範。
```

File: content/docs/getting-started.md [hunk 2]
```diff
@@ -18,109 +18,109 @@ redirect_from:
...
 
-### Add React to a Website {#add-react-to-a-website}
+### 將 React 加入到網頁 {#add-react-to-a-website}
 
-You can [add React to an HTML page in one minute](/docs/add-react-to-a-website.html). You can then either gradually expand its presence, or keep it contained to a few dynamic widgets.
+你可以[在一分鐘內將 React 加入到一個 HTML 網頁上](/docs/add-react-to-a-website.html)。你可以選擇逐步擴展它的應用範圍，或是只使用在少部分的可變 widget 上。
```
Comment (neighborhood999) [inline content/docs/getting-started.md]:
```suggestion
你可以[在一分鐘內將 React 加入到一個 HTML 網頁上](/docs/add-react-to-a-website.html)。你可以選擇逐步擴大它的應用範圍，或是只使用在少部分的可變 widget 上。
```

File: content/docs/getting-started.md [hunk 3]
```diff
@@ -18,109 +18,109 @@ redirect_from:
...
+我們假設你有一些 JavaScript 的基礎知識。你不需要是 JavaScript 的專家，只是同時學習 React 和 JavaScript 會比較困難。
 
-We recommend going through [this JavaScript overview](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level. It will take you between 30 minutes and an hour but you will feel more confident learning React.
+我們建議先看[這裏的 JavaScript 概覽](https://developer.mozilla.org/zh-TW/docs/Web/JavaScript/A_re-introduction_to_JavaScript)來測試自己的認識程度。這需要花費大概 30 分鐘到 1 小時，但這會增加你學習 React 的信心。
 
->Tip
+>提示
 >
->Whenever you get confused by something in JavaScript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) and [javascript.info](https://javascript.info/) are great websites to check. There are also [community support forums](/community/support.html) where you can ask for help.
+>當你遇到 JavaScript 的困難時，[MDN](https://developer.mozilla.org/zh-TW/docs/Web/JavaScript) 和 [javascript.info](https://javascript.info/) 會是非常好的查閱資源. 你也可以在[社群支援論壇](/community/support.html)上尋找幫助。
```
Comment (neighborhood999) [inline content/docs/getting-started.md]: The **MDN** link should be keeping **en**, thank you.
```suggestion
>當你遇到 JavaScript 的困難時，[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) 和 [javascript.info](https://javascript.info/) 會是非常好的查閱資源。你也可以在[社群支援論壇](/community/support.html)上尋找幫助。
```

File: content/docs/getting-started.md [hunk 4]
```diff
@@ -18,109 +18,109 @@ redirect_from:
...
+### 按部就班學習指南 {#step-by-step-guide}
 
-If you prefer to **learn concepts step by step,** our [guide to main concepts](/docs/hello-world.html) is the best place to start. Every next chapter in it builds on the knowledge introduced in the previous chapters so you won't miss anything as you go along.
+如果你喜歡**按部就班學習概念**，我們的[主要概念指南](/docs/hello-world.html)會是一個很好的起點。指南裏每個章節都是建基於上一個章節的學習內容，這樣你就不會錯過任何內容。
 
-### Thinking in React {#thinking-in-react}
+### 用 React 思考 {#thinking-in-react}
 
-Many React users credit reading [Thinking in React](/docs/thinking-in-react.html) as the moment React finally "clicked" for them. It's probably the oldest React walkthrough but it's still just as relevant.
+許多 React 用家都表示在閱讀了[用 React 思考](/docs/thinking-in-react.html)後對 React 恍然大悟。它可能是最源遠流長的 React 指南，至今依然極具價值。
```
Comment (neighborhood999) [inline content/docs/getting-started.md]:
```suggestion
許多 React 使用者都表示在閱讀了[用 React 思考](/docs/thinking-in-react.html)後對 React 恍然大悟。它可能是最源遠流長的 React 指南，至今依然極具價值。
```

**Comment by fabianlee1211** [comment]
@neighborhood999 glad to help the community out! thanks for the suggestions!

**Pull request merged**